src/components/sortableList/SortableListItem.driver.ts

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash';
+import {ComponentDriver} from '../../testkit/Component.driver';
+
+/**
+ * Please run clear after each test
+ * */
+export class SortableListItemDriver extends ComponentDriver {
+  dragUp = async (indices: number) => {
+    this.validateIndices(indices);
+    const data = _.times(indices, index => {
+      return {
+        translationY: -52 * (index + 1)
```

Comment:
Will it always be 52? 
The user can pass a different item height, no?

Reply:
It starts with 52 and then there's `onLayout` on the first item.
This does not happen in tests (at least for now).
It could change if we change the component (I rather have the tests fail than export a const from the code just for the tests).